lantern_hnsw/src/hnsw/build.c

```diff
@@ -486,6 +486,12 @@ static void BuildIndex(Relation heap, Relation index, IndexInfo *indexInfo, ldb_
 
     InitBuildState(buildstate, heap, index, indexInfo);
 
+    if(buildstate->index_file_path) {
+        elog(ERROR,
+             "Importing index from file is deprecated.\n"
```

Comment:
deprecated makes it sound like the functionality still exists, but we have removed the functionality already.
Rephrase to "Importing index from file is no longer supported"